### ISSUE TITLE
A very high memory usage in SQLite.

### ISSUE THREAD

**Issue by AlenToma** [expo/expo#16637]
### Summary

I am creating a global db and executing some operation with a background server after x ms time.

As time goes, the memory usage gets bigger and bigger for some reason.

After one day working the memory gets to 1 GB.

I have build an ORM [expo-sqlite-wrapper](https://www.npmjs.com/package/expo-sqlite-wrapper) and are using it to query data.

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

bare

### What platform(s) does this occur on?

Android

### SDK Version (managed workflow only)

44

### Environment

```
  expo-env-info 1.0.2 environment info:
    System:
      OS: Windows 10 10.0.19044
    Binaries:
      Node: 16.14.0 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.22.17 - ~\AppData\Roaming\npm\yarn.CMD
      npm: 8.3.1 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version     2021.1.0.0 AI-211.7628.21.2111.8193401
    npmPackages:
      expo: ^44.0.2 => 44.0.6
      react: 17.0.1 => 17.0.1
      react-dom: 17.0.1 => 17.0.1
      react-native: 0.64.3 => 0.64.3
    Expo Workflow: bare
```

### Reproducible demo

```
import * as SQLite from 'expo-sqlite';
import createDbContext, { IDatabase, IQueryResultItem, IBaseModule } from 'expo-sqlite-wrapper'
export default class Repository {
  static dbIni: Boolean;
  databaseName: string;
  database: IDatabase<TableNames>;
  constructor() {
    this.databaseName = 'test.db';
    this.database = createDbContext<TableNames>(tables, async () => SQLite.openDatabase(this.databaseName) as any);
  }

}
// this is a global things, so it stays open until the app is closed.
const rep = new Repository ();
// simple do select, save and so on
```

**Comment by AlenToma** [comment]
Ok I found the problem with this. I hade to apply those settings to make it work.
```
    PRAGMA cache_size=8192;
    PRAGMA encoding="UTF-8";
    PRAGMA synchronous=NORMAL;
    PRAGMA temp_store=FILE;
```

There should be some kind of documentation on expo documentation for this, So people should know that this issue could be a risk for them as there is no way to free or close the database manually.

**Comment by jonelofficial** [comment]
Hi @AlenToma  where did you applied the setting? I encountered the same please.

**Comment by AlenToma** [comment]
Simple exeute them when you create the open a dB.

Now you also have to close the connection when the app is ideal for x hour.

Those settings is applied under advanced example

**Comment by jonelofficial** [comment]
@AlenToma 

I saw the sample code on Expo Docs, is it correct?
`db.exec(
  [
    {
      sql: 'PRAGMA cache_size=8192; PRAGMA encoding="UTF-8"; PRAGMA synchronous=NORMAL; PRAGMA temp_store=FILE;',
      args: [],
    },
  ],
  false
);
`

**Comment by AlenToma** [comment]
> @AlenToma 
> 
> I saw the sample code on Expo Docs, is it correct?
> `db.exec(
>   [
>     {
>       sql: 
>       'PRAGMA cache_size=8192; 
>       PRAGMA encoding="UTF-8"; 
>       PRAGMA synchronous=NORMAL; 
>       PRAGMA temp_store=FILE;',
>       args: [],
>     },
>   ],
>   false
> );
> `
> 
> Could you please include an example? Sorry for the inconvenience; this is my first expo project. 

Yes it seems right to me. But also try to enable hermes as jsc tends to Increase the memory usage
